hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/BaseAPIFabricator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+abstract class CRM_Hrjobcontract_Test_Fabricator_BaseAPIFabricator {
+
+  protected static $default = [
+    'sequential' => 1
+  ];
+
+  /**
+   * Using the civicrm API and the create endpoint, fabricates a new entity
+   * with the given params.
+   *
+   * @param array $params
+   *  An array of params that will be passed to the civicrm API
+
+   * @return array
+   *  The entity values as they are returned by the API call
+   *
+   * @throws \Exception
+   */
+  public static function fabricate($params) {
+    if (!isset($params['jobcontract_id'])) {
+      throw new Exception('Specify jobcontract_id value');
+    }
+
+    $result = civicrm_api3(
+      static::getEntityName(),
+      'create',
+      array_merge(self::$default, $params)
+    );
+
+    return array_shift($result['values']);
```

Comment:
Please check Kacper's comment (https://github.com/civicrm/civihr/pull/1206#discussion_r83685653) and the commit message to see why array_shift is an better option